### PR TITLE
Guide for agent -> worker transition.

### PR DESCRIPTION
**Pull request by taylor-curran**
Issue: #9819 

I spend so so much time supporting people through this transition, I want to have a snappy guide in our docs for people that don't get as much 1-1 support.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# How to Upgrade from Agents to Workers
+
+Upgrading from agents to workers opens the door to a host of enhanced features. It's important to understand that some of these features are interconnected. For instance, to define your deployments in a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file, you must [create a Work Pool](/concepts/work-pools/#work-pool-configuration) and [start a worker](/concepts/work-pools/#starting-a-worker). This guide provides a comprehensive overview of the changes you'll encounter when upgrading from agents to workers.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:3]:
```suggestion
Upgrading from agents to workers significantly enhances the experience of deploying flows, especially with regards to specifying its infrastructure and runtime environment. This guide provides an overview of the differences between agents to workers and describes how to upgrade.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Enhancements
+
+**Agents to Workers**
+
+- Improved visibility into the status of the worker such as when workers were started and when they last polled.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:9]:
```suggestion
- Improved visibility into the status of each worker, including when a worker was started and when it last polled.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- Better handling of race conditions for high availability use cases.
+
+**Infra Blocks to Work Pools**
+
+- Work pools expose [`job_configuration`](/concepts/work-pools/#base-job-template) that enable an unprecedented level of customization and governance.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:14]:
```suggestion
- Work pools expose a [base job template](/concepts/work-pools/#base-job-template) that enables an unprecedented level of customization and governance over workers in that pool.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+Upgrading from agents to workers opens the door to a host of enhanced features. It's important to understand that some of these features are interconnected. For instance, to define your deployments in a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file, you must [create a Work Pool](/concepts/work-pools/#work-pool-configuration) and [start a worker](/concepts/work-pools/#starting-a-worker). This guide provides a comprehensive overview of the changes you'll encounter when upgrading from agents to workers.
+
+## Enhancements
+
+**Agents to Workers**
+
+- Improved visibility into the status of the worker such as when workers were started and when they last polled.
+- Better handling of race conditions for high availability use cases.
+
+**Infra Blocks to Work Pools**
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:12]:
```suggestion
**Agent Infra Blocks to Work Pools**
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+- Work pools expose [`job_configuration`](/concepts/work-pools/#base-job-template) that enable an unprecedented level of customization and governance.
+- New [push work pools](/guides/deployment/push-work-pools/) (beta) allow for flow execution without the need to host a worker.
+
+**New `prefect.yaml` file for managing multiple deployments**
+
+- Its easier than before to define many deployments en mass through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:19]:
```suggestion
- Its easier than before to define many multiple deployments at once through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+- Prefect provides [deployment actions](/concepts/deployments/#deployment-actions) that allow you to automatically build images for your flows.
+- [Templating](/concepts/deployments/#templating-options) enables [dryer deployment definitions](/concepts/deployments/#reusing-configuration-across-deployments).
+- You get a [wizard now](/#step-5-deploy-the-flow)! 🧙
+
+----------
+
+## What's changing
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:26]:
```suggestion
## What's different
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    run-infrastructure blocks --> [typed work pool](/concepts/work-pools/#worker-types) 
+    
+    Infra config is now set on the typed Work Pool.
+
+4. **Managing multiple deployments:**
+    
+    Create and/or update many deployments en mass through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
+
+
+## What stays the same
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:51]:
```suggestion
## What's the same
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+- The process for starting an agent and [starting a worker](/concepts/work-pools/#starting-a-worker) in your environment are virtually identical.
+    
+    `prefect agent start <work pool>` --> `prefect worker start <work pool>`
+
+
+## How to get started quickly
+
+If you have a deployment with a storage block and infra block, here's how to quickly upgrade it to use Prefect's new features:
+
+1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of type that matches whichever run-infrastructure block you are currently using.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:68]:
```suggestion
1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of the same type as the infrastructure block you are currently using.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+    `prefect agent start <work pool>` --> `prefect worker start <work pool>`
+
+
+## How to get started quickly
+
+If you have a deployment with a storage block and infra block, here's how to quickly upgrade it to use Prefect's new features:
+
+1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of type that matches whichever run-infrastructure block you are currently using. 
+
+    1. Any work pool infrastructure type aside from `Prefect Agent` should work.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:70]:
```suggestion
    1. Any work pool infrastructure type other than `Prefect Agent` will work.
```

File: mkdocs.yml [hunk 10]
```diff
@@ -179,6 +179,7 @@ nav:
           - Automations: concepts/automations.md
           - Filesystems: concepts/filesystems.md
           - Block and Agent-Based Deployments:
+                - Upgrading from Agents to Workers: guides/upgrade-guide-agents-to-workers.md
```
Comment (billpalombi) [inline mkdocs.yml:182]:
```suggestion
                - Upgrade from Agents to Workers: guides/upgrade-guide-agents-to-workers.md
```

**Review by billpalombi — COMMENTED** [review]
Thanks for making this! This will be really helpful.

I've got a bunch of pedantic comments but thats only because I think this is an important resource.

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
+# How to Upgrade from Agents to Workers
+
+Upgrading from agents to workers opens the door to a host of enhanced features. It's important to understand that some of these features are interconnected. For instance, to define your deployments in a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file, you must [create a Work Pool](/concepts/work-pools/#work-pool-configuration) and [start a worker](/concepts/work-pools/#starting-a-worker). This guide provides a comprehensive overview of the changes you'll encounter when upgrading from agents to workers.
+
+## Enhancements
+
+**Agents to Workers**
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:7]:
```suggestion
### Agents to workers
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+- Improved visibility into the status of the worker such as when workers were started and when they last polled.
+- Better handling of race conditions for high availability use cases.
+
+**Infra Blocks to Work Pools**
+
+- Work pools expose [`job_configuration`](/concepts/work-pools/#base-job-template) that enable an unprecedented level of customization and governance.
+- New [push work pools](/guides/deployment/push-work-pools/) (beta) allow for flow execution without the need to host a worker.
+
+**New `prefect.yaml` file for managing multiple deployments**
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:17]:
```suggestion
### New `prefect.yaml` file for managing multiple deployments
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+3. **Configuring flow run infrastructure:** 
+    
+    run-infrastructure blocks --> [typed work pool](/concepts/work-pools/#worker-types) 
+    
+    Infra config is now set on the typed Work Pool.
+
+4. **Managing multiple deployments:**
+    
+    Create and/or update many deployments en mass through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:48]:
```suggestion
    Create and/or update many deployments at once through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of the same type as the infrastructure block you are currently using.
+
+    1. Any work pool infrastructure type other than `Prefect Agent` will work.
+    2. Referencing the configurations you've set on the infrastructure block, set similar flow run infrastructure configurations on the work pool.
+
+2. [Start a worker](/concepts/work-pools/#starting-a-worker) to poll this work pool. You should see the command to start the worker as soon as you save your new work pool. 
+
+    !!! Tip "Process for starting a worker is very similar to the process for starting a worker."
+        Notice the command to start a worker is very similar to the command to start an agent. If you previously used terraform, a helm chart, or other infra as code method to start your agent, you should be able to continue using it for the worker provided the word `agent` is changed to `worker`.
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:76]:
```suggestion
        Notice the command to start a worker is very similar to the command to start an agent. If you previously used terraform, a helm chart, or other infrastructure-as-code method to start an agent, you should be able to continue using it for a worker, provided the word `agent` is changed to `worker`.
```

**Review by discdiver — COMMENTED** [review]
Cool. Found a few changes.

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 15]
```diff
@@ -0,0 +1,87 @@
+# How to Upgrade from Agents to Workers
+
+Upgrading from agents to workers significantly enhances the experience of deploying flows, especially with regards to specifying its infrastructure and runtime environment. This guide provides an overview of the differences between agents and workers and describes how to upgrade.
+
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:4]:
```suggestion
A worker is the fusion of an [agent](/concepts/agents/) with an [infrastructure block](/concepts/infrastructure/). Like agents, they poll a work pool for submitted flow runs that are scheduled to start. Like infrastructure blocks, they are typed - they work with only one kind of infrastructure and they specify the configuration of that infrastructure.

Accordingly, workers are not a drop in replacement for agents. Using workers requires deploying flows differently. In particular, deploying a flow with an agent does not involve specifying an infrastructure block. Instead, infrastructure configuration is specified on the [work pool](/concepts/work-pools/) and passed to each worker that polls work from that pool.

```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+4. **Managing multiple deployments:**
+    
+    Create and/or update many deployments at once through a [`prefect.yaml`](/concepts/deployments/#managing-deployments) file.
+
+
+## What's the same
+
+- Storage blocks can be set as the pull action in a `prefect.yaml` file.
+- Infra blocks have similar configuration fields as typed work pools.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:54]:
```suggestion
- Infrastructure blocks have similar configuration fields as typed work pools.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+
+
+## How to get started quickly
+
+If you have a deployment with a storage block and infra block, here's how to quickly upgrade it to use Prefect's new features:
+
+1. [Create a work pool](/concepts/work-pools/#work-pool-configuration) of the same type as the infrastructure block you are currently using.
+
+    1. Any work pool infrastructure type other than `Prefect Agent` will work.
+    2. Referencing the configurations you've set on the infrastructure block, set similar flow run infrastructure configurations on the work pool.
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:71]:
```suggestion
    2. Referencing the configuration you've set on the infrastructure block, set similar flow run infrastructure configuration on the work pool.
```

**Review by billpalombi — COMMENTED** [review]
Another pass with a brief description up top.

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+    `infra_override` -> [`job_variable`](/concepts/deployments/#work-pool-fields)
+
+- The process for starting an agent and [starting a worker](/concepts/work-pools/#starting-a-worker) in your environment are virtually identical.
+    
+    `prefect agent start <work pool>` --> `prefect worker start <work pool>`
+
+
+## How to get started quickly
+
+If you have a deployment with a storage block and infra block, here's how to quickly upgrade it to use Prefect's new features:
```
Comment (billpalombi) [inline docs/guides/upgrade-guide-agents-to-workers.md:70]:
```suggestion
If you have existing deployments that use infrastructure blocks, you can quickly upgrade them to be compatible with workers by:
```

**Comment by taylor-curran** [comment]
I think the descriptions is a bit repetitive, i can try to clean it up after pacc

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
+# How to Upgrade from Agents to Workers
+
+Upgrading from agents to workers significantly enhances the experience of deploying flows. It simplifies the specification of each flow's infrastructure and runtime environment. 
+
+A worker is the fusion of an [agent](/concepts/agents/) with an [infrastructure block](/concepts/infrastructure/). Like agents, they poll a work pool for submitted flow runs that are scheduled to start. Like infrastructure blocks, they are typed - they work with only one kind of infrastructure and they specify the configuration of that infrastructure.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:5]:
```suggestion
A worker is the fusion of an [agent](/concepts/agents/) with an [infrastructure block](/concepts/infrastructure/). Like agents, workers poll a work pool for flow runs that are scheduled to start. Like infrastructure blocks, workers are typed - they work with only one kind of infrastructure and they specify the default configuration for jobs submitted to that infrastructure.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+
+Accordingly, workers are not a drop in replacement for agents. **Using workers requires deploying flows differently.** In particular, deploying a flow with an worker does not involve specifying an infrastructure block. Instead, infrastructure configuration is specified on the [work pool](/concepts/work-pools/) and passed to each worker that polls work from that pool.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:7]:
```suggestion
Accordingly, workers are not a drop-in replacement for agents. **Using workers requires deploying flows differently.** In particular, deploying a flow with a worker does not involve specifying an infrastructure block. Instead, infrastructure configuration is specified on the [work pool](/concepts/work-pools/) and passed to each worker that polls work from that pool.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+## Enhancements
+
+### Agents to workers
+
+- Improved visibility into the status of each worker, including when a worker was started and when it last polled.
+- Better handling of race conditions for high availability use cases.
+
+### Agent infrastructure blocks to work pools
+
+- Work pools expose a [base job template](/concepts/work-pools/#base-job-template) that enables an unprecedented level of customization and governance over workers in that pool.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:20]: "...and governance" 💯

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+- [Push work pools](/guides/deployment/push-work-pools/) allow for flow execution without the need to host a worker.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:21]:
```suggestion
- [Push work pools](/guides/deployment/push-work-pools/) allow for flow execution without the need to host a worker at all.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+
+1. **Command to build deployments:** 
+    
+    `prefect deployment build <entrypoint>` --> [`prefect deploy`](/concepts/deployments/#deployment-declaration-reference) 
+    
+    Prefect will now automatically detect flows in your repo and provide a [wizard](/#step-5-deploy-the-flow) 🧙 to guide you through setting required attributes for your deployments.
+
+2. **Configuring remote flow code storage:** 
+    
+    storage blocs --> [pull action](/concepts/deployments/#the-pull-action)
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:42]:
```suggestion
    storage blocks --> [pull action](/concepts/deployments/#the-pull-action)
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 24]
```diff
@@ -0,0 +1,93 @@
...
+    
+    Though you can still use an existing [storage block as your pull action](/guides/deployment/storage-guide/)!
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:44]:
```suggestion
    And you can still use an existing [storage block as your pull action](/guides/deployment/storage-guide/)!  The more general pull steps specification allows for explicit customization of the job that defines each run (for example, by adding an additional step that sets a working directory).
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 25]
```diff
@@ -0,0 +1,93 @@
...
+
+3. **Configuring flow run infrastructure:** 
+    
+    run-infrastructure blocks --> [typed work pool](/concepts/work-pools/#worker-types) 
+    
+    Infra config is now set on the typed work pool.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:50]:
```suggestion
    Default infra config is now set on the typed work pool, and can be overwritten by both individual deployments or individual runs.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 26]
```diff
@@ -0,0 +1,93 @@
...
+    `infra_override` -> [`job_variable`](/concepts/deployments/#work-pool-fields)
+
+- The process for starting an agent and [starting a worker](/concepts/work-pools/#starting-a-worker) in your environment are virtually identical.
+    
+    `prefect agent start <work pool>` --> `prefect worker start <work pool>`
+
+
+## How to upgrade quickly
+
+If you have existing deployments that use infrastructure blocks, you can quickly upgrade them to be compatible with workers by:
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:72]:
```suggestion
If you have existing deployments that use infrastructure blocks, you can quickly upgrade them to be compatible with workers by following these steps:
```
(added this to match the voice of the steps)

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 27]
```diff
@@ -0,0 +1,93 @@
...
+    2. Referencing the configuration you've set on the infrastructure block, set similar flow run infrastructure configuration on the work pool.
+
+2. [Start a worker](/concepts/work-pools/#starting-a-worker) to poll this work pool. You should see the command to start the worker as soon as you save your new work pool. 
+
+    ```
+    prefect worker start -p [work pool name]
+    ```
+
+    !!! Tip "Process for starting a worker is very similar to the process for starting an agent"
+        Notice the command to start a worker is very similar to the command to start an agent. If you previously used terraform, a helm chart, or other infrastructure-as-code method to start an agent, you should be able to continue using it for a worker, provided the word `agent` is changed to `worker`.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:86]:
```suggestion
        Notice the command to start a worker is very similar to the command to start an agent. If you previously used terraform, a helm chart, or other infrastructure-as-code methods to start an agent, you should be able to continue using it for a worker, provided all uses of `agent` are changed to `worker`.
```

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 28]
```diff
@@ -0,0 +1,93 @@
...
+
+3. [Deploy your flow](/concepts/deployments/#deployment-mechanics):
+    ```bash
+    prefect deploy
+    ```
+4. In your prefect.yaml file, configure a [pull action](/guides/deployment/storage-guide/) referencing whatever configuration you used as your storage block.
```
Comment (cicdw) [inline docs/guides/upgrade-guide-agents-to-workers.md:92]:
```suggestion
4. In your `prefect.yaml` file, configure a [pull action](/guides/deployment/storage-guide/) referencing whatever configuration you used as your storage block.
```

**Review by cicdw — COMMENTED** [review]
awesome thank you for this! I made a bunch of small suggestions, otherwise LGTM

**Review by billpalombi — APPROVED** [review]
I think this looks good to go

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 29]
```diff
@@ -0,0 +1,96 @@
...
+
+- Storage blocks can be set as the pull action in a `prefect.yaml` file.
+- Infrastructure blocks have similar configuration fields as typed work pools.
+- Deployment-level infra-overrides operate in much the same way. 
+
+    `infra_override` -> [`job_variable`](/concepts/deployments/#work-pool-fields)
+
+- The process for starting an agent and [starting a worker](/concepts/work-pools/#starting-a-worker) in your environment are virtually identical.
+    
+    `prefect agent start <work pool>` --> `prefect worker start <work pool>`
```
Comment (zzstoatzz) [inline docs/guides/upgrade-guide-agents-to-workers.md:67]:
```suggestion
    `prefect agent start --pool <work pool name>` --> `prefect worker start --pool <work pool name>`
```

**Review by zzstoatzz — APPROVED** [review]

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 30]
```diff
@@ -0,0 +1,101 @@
...
+    run-infrastructure blocks --> [typed work pool](/concepts/work-pools/#worker-types) 
+    
+    Default infra config is now set on the typed work pool, and can be overwritten by both individual deployments or individual runs.
+
+4. **Managing multiple deployments:**
+    
+    Create and/or update many deployments at once through a [`prefect.yaml`](/concepts/deployments/#working-with-multiple-deployments) file.
+
+
+## What's the same
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:57]:
```suggestion
## What's similar
```

Comment (discdiver) [reply docs/guides/upgrade-guide-agents-to-workers.md:57]: Most of these things are similar, but not the same

**Review by discdiver — APPROVED** [review]
LGTM! One minor change suggested.

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 31]
```diff
@@ -0,0 +1,101 @@
...
+
+2. [Start a worker](/concepts/work-pools/#starting-a-worker) to poll this work pool. You should see the command to start the worker as soon as you save your new work pool. 
+
+    ```
+    prefect worker start -p <work pool name>
+    ```
+
+3. [Deploy your flow](/#step-5-deploy-the-flow) guided by the deployment creation wizard:
+
+    !!! warning "Always run `prefect deploy` commands from the **root** level of your repo!"
```
Comment (taylor-curran) [inline docs/guides/upgrade-guide-agents-to-workers.md:89]: Could I get your eyes on the wording of this warning and note block?

@discdiver

File: docs/guides/upgrade-guide-agents-to-workers.md [hunk 32]
```diff
@@ -0,0 +1,101 @@
...
+        With agents you might have had multiple `deployment.yaml` files, but under worker deployment patterns, each repo will have a single `prefect.yaml` file located at the **root** of the repo that contains a [list of each deployment](/concepts/deployments/#working-with-multiple-deployments) for all flows in that repo.
```
Comment (discdiver) [inline docs/guides/upgrade-guide-agents-to-workers.md:90]:
```suggestion
        With agents you might have had multiple `deployment.yaml` files, but under worker deployment patterns, each repo will have a single `prefect.yaml` file located at the **root** of the repo that contains [deployment configuration](/concepts/deployments/#working-with-multiple-deployments) for all flows in that repo.
```

**Review by discdiver — APPROVED** [review]
Nice!

**Pull request merged**